CHANGELOG.md

```diff
@@ -1,15 +1,24 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This project adheres to [CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
 
-## [0.1.1] - 2916-03-17
-- fixed hard coded label mapping not matching with prefix 'thumbor_'
+## [1.0.0] - 2022-07-01
+
+- Python 3 and Thumbor 7 support
+- Removed pyvows in favor of pytest
+
+## [0.1.1] - 2017-03-17
+
+- fixed hard coded label mapping not matching with prefix 'thumbor\_'
 
 ## [0.1.0] - 2017-02-23
+
 ### Added
+
```

Comment:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).